content/docs/rfcs/23/README.md

```diff
@@ -65,30 +83,25 @@ Let's say we have two different topics that are both experience heavy traffic bu
 This can be segregated into:
 
 ```
-/waku/2/status/proto
-/waku/2/walletconnect/proto
+/waku/2/status/
+/waku/2/walletconnect/
 ```
 
 This indicates that they are WakuMessages but for different domains completely.
 
 ### Topic sharding example
 
-Topic sharding is currently not supported by default, but is planned for the future in order to deal with increased network traffic.
-Here's an example of what this might look like:
+The following is an example of named sharding, as specified in [51/WAKU2-RELAY-SHARDING](/spec/51).
 
 ```
-waku/2/waku-9_shard-0/proto
+waku/2/waku-9_shard-0/
 ...
-waku/2/waku-9_shard-9/proto
+waku/2/waku-9_shard-9/
 ```
 
 This indicates explicitly that the network traffic has been partitioned into 10 buckets.
 
-### Compression example
-
-Not yet implemented, but would be easy to add with:
-
-`/waku/2/default-waku/proto_snappy`
+Besides named sharding, [51/WAKU2-RELAY-SHARDING](/spec/51) specifies two more sharding methods: static sharding and automatic sharding.
```

Comment:
This section does not convince me because this is referencing something that references this RFC (I see a cyclic dependency 😅). I would not specify anything about sharding in RFC 23.

Reply:
Additionally, you have already provided some examples above, `rs/0/2`. So I would leave this part for RFC 51.

Reply:
(2) makes the most sense.

I've been trying to find specific guidelines on cyclic dependencies and cross-referencing in RFCs when evolving a concept. This of course cannot be done for normative references, but IMO, since this is not a normative RFC and `51` is not a normative reference, I don't have a _major_ problem to have an appendix/section which is not part of the main text that briefly points people to the evolution of ideas in this RFC in `51`. It makes navigating to the latest "full picture" a bit easier.

Reply:
Mmm, I had more in mind simplifying this whole section to a line or two in an appendix ("Future work"?) with possible reference to `51` for easier navigation (but, in other words, the specification for named sharding and default pubsub topic happens there, not in `23`). But I don't have a strong opinion here, because we tend to be more pragmatic when it's not a normative spec. Might make sense to just do the simplest thing here for now.